src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx

```diff
@@ -35,38 +35,39 @@ To install the infrastructure monitoring agent, follow the step-by-step instruct
 2. Make sure [Homebrew](https://brew.sh/) is installed in the system.
     You can check your Homebrew installation with:
 
-    ```
+    ```bash
       which brew
     ```
 
     If it's not installed, you can install it with this command (or check [Homebrew](https://brew.sh/) up-to-date instructions):
 
-    ```
+    ```bash
       /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
     ```
 
-3. Then, open the terminal and run the following command:
+3. In your terminal, run `mkdir -p` to create a new directory for the infrastructure agent, then run the following command:
```

Comment:
Added the mkdir command together with the file creation below, we can dismiss this change.